e2etest/zt_newe2e_oauth_device_test.go

```diff
@@ -0,0 +1,118 @@
+package e2etest
+
+import (
+	"flag"
+	"github.com/Azure/azure-storage-azcopy/v10/common"
+	"github.com/Azure/azure-storage-azcopy/v10/testSuite/cmd"
+	"path"
+	"strings"
+	"time"
+)
+
+var runDeviceCodeTest = flag.Bool("device-code", false, "Whether or not to run device code tests. These must be run manually due to interactive nature.")
+
+var manualEnv AzCopyEnvironment
+
+func init() {
+	if runDeviceCodeTest != nil && *runDeviceCodeTest {
+		suiteManager.RegisterSuite(&DeviceLoginManualSuite{})
+	}
+}
+
+func Scenario_CopySync(svm *ScenarioVariationManager, env *AzCopyEnvironment) {
```

Comment:
env also is unused and this has a code smell about it

If the goal were to set up a consistent environment for the suite, the best idea would be to add it as a field under the suite struct, but between a global function and a global variable, this feels like an odd implementation.

That said, neither of these wind up getting used, so, it should get removed.

Reply:
this is accidental leftover from some of my debugging. will remove.